src/index.js

```diff
@@ -1,8 +1,21 @@
+import mapboxgl from 'mapbox-gl'
+import MapboxGeocoder from 'mapbox-gl-geocoder'
+import Config from './config.js'
+import _ from 'lodash'
+import Filter from './js/filter.js'
+
+// Import Styles
+import 'mapbox-gl/dist/mapbox-gl.css'
+import 'mapbox-gl-geocoder/dist/mapbox-gl-geocoder.css'
+import './styles/normalize.css'
+import './styles/styles.css'
+
 const $locationList = document.getElementById('location-list')
 const $sidePane = document.getElementById('side-pane')
 const $locationsButton = document.getElementById('locations-toggle-button');
```

Comment:
If we're no longer using this then we should be able to remove the variable definition